examples/with-tailwind/apps/web/package.json

```diff
@@ -1,16 +1,18 @@
 {
   "name": "web",
   "version": "1.0.0",
+  "type": "module",
```

Comment:
For eslint flat?

Reply:
It will break out generators, so would prefer we keep it out until we have that migrated.

Reply:
Not locked to it since we don't have one here. But could cause more issues for us until we update that package